gradle/dependency-versions.gradle

```diff
@@ -47,7 +47,8 @@
   scalaTestVersion = "3.0.1"
   snappyVersion = "1.1.8.4"
   slf4jVersion = "1.7.7"
-  yarnVersion = "2.10.1"
```

Comment:
Is it possible to split up the Hadoop version bump (and the related hadoop zookeeper gradle exclude) to a separate PR? It feels like it may be significant enough to stand alone.

Reply:
Updated in the next push